Cargo.toml

```diff
@@ -88,5 +88,6 @@ It is heavily inspired by ncurses MPD clients, such as ncmpc."""
 license-file = ["LICENSE"]
 assets = [
     ["target/release/ncspot", "usr/bin/", "755"],
+    ["misc/ncspot.desktop", "usr/share/applications/", "644"],
```

Comment:
I'm not very familiar with the debian packaging manifest, so I'm not sure if this is good enough.